setup.py

```diff
@@ -10,7 +10,6 @@
 # binary wheels.
 
 import copy
-from distutils.command.sdist import sdist
```

Comment:
unrelated to PR goal: sdist is not used in setup.py

Reply:
Also good because `distutils` is deprecated: https://www.python.org/dev/peps/pep-0632/